content/aips/USDTETHV3-AIP.md

```diff
@@ -0,0 +1,92 @@
+---
+title: USDTethV3-AIP
+author: Marc Zeller (@marczeller), Aave-Chan Inititative
+shortDescription: Onboard USDT Aave Eth V3
+discussions: https://governance.aave.com/t/arfc-add-usdt-to-ethereum-v3-market/11536
+created: 2023-02-07
+updated: 2023-02-07
+---
+
+## Simple Summary
+This AIP proposal presents Aave with the opportunity to onboard USDT to the Ethereum v3 Liquidity Pool.
+
+## Motivation
+Supporting stablecoin diversity is part of the Aave-Chan Initiative (ACI) delegate platform.cd ..
+
+The ACI does not have any link nor is paid by Tether or any entity to present this ARC.
+
+The risk parameters provided introduce USDT as a non-collateral asset.
+
+The Interest rate Strategy curve is the same as for USDT in V2. for now.
+The ACI is supportive of onboarding USDT as it was in V2.
+For now, the ACI doesn’t consider V3 mature enough to activate a USD stablecoins emode and consider USDT in it.
+Also, supply and borrow caps are suggested for this asset to limit risk exposure.
+
+## Specification
+
+Ticker: USDT
+
+Contract Address: 0xdAC17F958D2ee523a2206206994597C13D831ec7
+
+|Risk Parameter|Value|
+| --- | --- |
+|Isolation Mode|NO|
+|Enable Borrow|YES|
+|Enable Collateral|NO|
+|Loan To Value|N/A|
+|Liquidation Threshold|N/A|
+|Liquidation Bonus|N/A|
+|Reserve Factor|10%|
+|Liquidation Protocol Fee|0.1|
+|Borrow Cap|185M|
+|Supply Cap|200M|
+|Debt Ceiling|185M|
```

Comment:
This should be N/A as is configured on the Payload, as USDT will not be added to Isolation mode